### PR TITLE
Improve markdown files

### PR DESCRIPTION
**Pull request by raballew**
The following changes have been applied:

- Use a maximum line length of 120 chars
- Fix several typos
- Improve readability of doc/test/hirte_e2e.md
- Improve readability of project description
- Remove stale file doc/test/orch_test_notes
- Use same spelling for Hirte

### CODE REVIEW

**Review by Yarboa — APPROVED** [review]
LGTM

File: README.developer.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ sudo dnf config-manager --set-enabled crb
 sudo dnf install clang-tools-extra gcc make meson systemd-devel
 ```
 
-[markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) can be used for static analysis of
-markdown files. Unfortunately it's not available as RPM, so please check its
-[installation guide](https://github.com/DavidAnson/markdownlint-cli2#install) and use the most appropriate way of
-installation for your setup.
+[markdownlint-cli2](https://github.com/DavidAnson/markdownlint-cli2) can be used for static analysis of markdown files.
+Unfortunately it's not available as RPM, so please check its [installation
```
Comment (mwperina) [inline README.developer.md]: Personally I'd leave URL links on a single line rather than having new line inside

File: README.md [hunk 2]
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
...
 
-Please submit patches to [github.com:containers/hirte](https://github.com/containers/hirte).
-More information about the development can be found in [README.developer.md](README.developer.md).
+Please submit patches to [github.com/containers/hirte](https://github.com/containers/hirte). More information about the
+development can be found in [README.developer.md](README.developer.md).
 
-If you are not familiar with the development process you can read about it in
-[Get started with GitHub](https://docs.github.com/en/get-started).
+If you are not familiar with the development process you can read about it in [Get started with
+GitHub](https://docs.github.com/en/get-started).
```
Comment (mwperina) [inline README.md]: Personally I'd leave URL links on a single line rather than having new line inside

File: README.md [hunk 3]
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
...
 
 ### Found a bug or documentation issue?
 
-To submit a bug or suggest an enhancement for hirte please use
-[GitHub issues](https://github.com/containers/hirte/issues).
+To submit a bug or suggest an enhancement for Hirte please use [GitHub
+issues](https://github.com/containers/hirte/issues).
```
Comment (mwperina) [inline README.md]: Personally I'd leave URL links on a single line rather than having new line inside

File: README.md [hunk 4]
```diff
@@ -16,18 +17,18 @@ to generate systemd service configuration to run a container.
...
 
 ## Still need help?
 
-If you have any other questions, please join
-[CentOS Automotive SIG mailing list](https://lists.centos.org/mailman/listinfo/centos-automotive-sig/) and ask there.
+If you have any other questions, please join [CentOS Automotive SIG mailing
+list](https://lists.centos.org/mailman/listinfo/centos-automotive-sig/) and ask there.
```
Comment (mwperina) [inline README.md]: Personally I'd leave URL links on a single line rather than having new line inside

**Review by mwperina — COMMENTED** [review]
Other than splitting URL over multiple lines it looks good to me

**Review by mwperina — APPROVED** [review]
+1

**Review by engelmi — APPROVED** [review]

**Pull request merged**